tests/approvals/test_notify_pole_emploi.py

```diff
@@ -337,10 +338,227 @@ def test_notification_use_origin_values(self):
         )
 
 
+@override_settings(
+    API_ESD={
+        "BASE_URL": "https://pe.fake",
+        "AUTH_BASE_URL": "https://auth.fr",
+        "KEY": "foobar",
+        "SECRET": "pe-secret",
+    }
+)
+@freeze_time("2021-06-21")
```

Comment:
J'ai pas l'impression que les tests ont besoin de cette date exactement
```suggestion
@freeze_time()
```